app/Language/fr/Community.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Community page messages for the CodeIgniter website
+ * The text blocks use markdown format.
+ */
+$securityPlug = <<<'EOS'
+Les problèmes de sécurité doivent être signalés avec un courrier électronique à notre 
+[security team](mailto: security@codeigniter.com), plutôt 
+que d'être soulevés sur le forum ou soulevés comme un problème Github, merci!
+EOS;
+
+$forumCommunity = <<<'EOS'
+Notre forum a été configuré avec MyBB, avec cinq catégories principales:
+
+- Général (actualités et discussion, salons, événements et groupes d'utilisateurs régionaux)
+
+- Utilisation de CodeIgniter (choix de CodeIgniter, installation et paramétrage, contrôleur modèle-vue-contrôleur, bibliothèques et assistants, meilleures pratiques et aide générale)
+
+- CodeIgniter 4 (feuille de route, développement, demandes de fonctionnalités, support, discussion)
+
+- Développement (CodeIgniter 3.x, CodeIgniter 2.x et problèmes)
+
+- Ressources externes (compléments, emplois, en savoir plus et sous les projecteurs)
+
+** Le forum est l'endroit où vous pouvez demander de l'aide ou discuter des problèmes que vous rencontrez avec framework. **
+
+
+<ul class="nav nav-pills justify-content-center">
+	<li class="nav-item" style="margin:0.5em;">
+		<a class="nav-link btn-primary" href="{mybb_forum_url}"><i class="glyphicon glyphicon-share"></i> {forumButton}</a>
+	</li>
+</ul>
+EOS;
+
+$slackCommunity = <<<'EOS'
+CodeIgniter a un serveur Slack, où vous pouvez dialoguer avec d'autres membres de la communauté.
+Tout le monde peut [s'inscrire] (http://codeignitersignup.ciblox.com/) pour cela :)
```

Comment:
Can't have space between [...] (...) - that breaks the link in generated text on page. That is markdown syntax.